2021-12-13 18:11:59,795 INFO    MainThread:1168 [wandb_setup.py:_flush():71] setting env: {}
2021-12-13 18:11:59,795 INFO    MainThread:1168 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-13 18:11:59,796 INFO    MainThread:1168 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20211213_181159-2fjjiiqz\logs\debug.log
2021-12-13 18:11:59,796 INFO    MainThread:1168 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20211213_181159-2fjjiiqz\logs\debug-internal.log
2021-12-13 18:11:59,797 INFO    MainThread:1168 [wandb_init.py:init():404] calling init triggers
2021-12-13 18:11:59,798 INFO    MainThread:1168 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-12-13 18:11:59,800 INFO    MainThread:1168 [wandb_init.py:init():449] starting backend
2021-12-13 18:11:59,800 INFO    MainThread:1168 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2021-12-13 18:11:59,811 INFO    MainThread:1168 [backend.py:ensure_launched():199] starting backend process...
2021-12-13 18:12:00,011 INFO    MainThread:1168 [backend.py:ensure_launched():204] started backend process with pid: 11252
2021-12-13 18:12:00,013 INFO    MainThread:1168 [wandb_init.py:init():458] backend started and connected
2021-12-13 18:12:00,016 INFO    MainThread:1168 [wandb_init.py:init():519] updated telemetry
2021-12-13 18:12:00,064 INFO    MainThread:1168 [wandb_init.py:init():549] communicating current version
2021-12-13 18:12:01,001 INFO    MainThread:1168 [wandb_init.py:init():554] got version response 
2021-12-13 18:12:01,002 INFO    MainThread:1168 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-13 18:12:01,339 INFO    MainThread:1168 [wandb_init.py:init():592] starting run threads in backend
2021-12-13 18:12:01,494 INFO    MainThread:1168 [wandb_run.py:_console_start():1816] atexit reg
2021-12-13 18:12:01,495 INFO    MainThread:1168 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2021-12-13 18:12:01,496 INFO    MainThread:1168 [wandb_run.py:_redirect():1727] Wrapping output streams.
2021-12-13 18:12:01,500 INFO    MainThread:1168 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-13 18:12:01,500 INFO    MainThread:1168 [wandb_init.py:init():619] run started, returning control to user process
2021-12-13 18:12:01,501 INFO    MainThread:1168 [wandb_config.py:__setitem__():144] config set learning_rate = 0.001 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000002A123B78B50>>
2021-12-13 18:12:01,501 INFO    MainThread:1168 [wandb_run.py:_config_callback():962] config_cb learning_rate 0.001 None
2021-12-13 18:12:01,503 INFO    MainThread:1168 [wandb_config.py:__setitem__():144] config set batch_size = 10 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000002A123B78B50>>
2021-12-13 18:12:01,503 INFO    MainThread:1168 [wandb_run.py:_config_callback():962] config_cb batch_size 10 None
2021-12-13 18:12:03,781 INFO    MainThread:1168 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-12-13 18:12:03,782 INFO    MainThread:1168 [wandb_run.py:_restore():1758] restore
2021-12-13 18:12:04,063 INFO    MainThread:1168 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-12-13 18:12:04,368 INFO    MainThread:1168 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-12-13 18:12:05,368 INFO    MainThread:1168 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 4873
}

2021-12-13 18:12:05,474 INFO    MainThread:1168 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 4899
}

2021-12-13 18:12:05,591 INFO    MainThread:1168 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 4899
}

2021-12-13 18:12:05,703 INFO    MainThread:1168 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2232
  total_bytes: 4899
}

2021-12-13 18:12:05,813 INFO    MainThread:1168 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4899
  total_bytes: 4899
}

2021-12-13 18:12:05,931 INFO    MainThread:1168 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4899
  total_bytes: 4899
}

2021-12-13 18:12:06,054 INFO    MainThread:1168 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4899
  total_bytes: 4899
}

2021-12-13 18:12:06,168 INFO    MainThread:1168 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4899
  total_bytes: 4899
}

2021-12-13 18:12:06,435 INFO    MainThread:1168 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4899
  total_bytes: 4899
}

2021-12-13 18:12:07,011 INFO    MainThread:1168 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4899
  total_bytes: 4899
}
local_info {
}

2021-12-13 18:12:08,108 INFO    MainThread:1168 [wandb_run.py:_append_files():2184] logging synced files
